src/components/ProjetsCards.jsx

```diff
@@ -20,7 +22,7 @@ export default function ProjetsCards() {
 
   useEffect(() => {
     axios
-      .get("http://localhost:5050/projets")
+      .get(`${api.defaults.baseURL}/projets`)
```

Comment:
I may be wrong, but this syntax is lighter and should work:
```
axios.get("/logos/")
```I may be wrong, but this syntax is lighter and should work:
```
axios.get("/logos/")
```